test/server_test.lua

```diff
@@ -19,6 +19,14 @@ local server = Server:new({
     net_box_port = 3133,
 })
 
+local SOCKET_DIR = require('fio').cwd()
+local server_with_socket = Server:new({
+    command = command,
+    workdir = fio.pathjoin(datadir, 'common_socket'),
+    net_box_uri = SOCKET_DIR .. "/test_socket.sock",
```

Comment:
What about setting it equal to `fio.pathjoin(datadir, '/test_socket.sock')` ? Without `SOCKET_DIR` at all.